recaptcha_enterprise/snippets/create_mfa_assessment.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START recaptcha_enterprise_mfa_assessment]
+
+from google.cloud import recaptchaenterprise_v1
+from google.cloud.recaptchaenterprise_v1 import Assessment
+
+
+def create_mfa_assessment(
+    project_id: str, recaptcha_site_key: str, token: str, recaptcha_action: str, hashed_account_id: str,
+        email: str, phone_number: str
+) -> None:
+    """  Creates an assessment to obtain Multi-Factor Authentication result.
+         If the result is unspecified, sends the request token to the caller to initiate MFA challenge.
+    Args:
+        project_id: GCloud Project ID
+        recaptcha_site_key: Site key obtained by registering a domain/app to use recaptcha services.
+        token: The token obtained from the client on passing the recaptchaSiteKey.
+               To get the token, integrate the recaptchaSiteKey with frontend. See,
+               https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+        recaptcha_action: The action name corresponding to the token.
+        hashed_account_id: Create hashedAccountId from user identifier.
+                           It's a one-way hash of the user identifier: HMAC SHA 256 + salt
+        email: Email id of the user to trigger the MFA challenge.
+        phone_number: Phone number of the user to trigger the MFA challenge.
```

Comment:
The spacing is consistent but a bit odd
```suggestion
        project_id: GCloud Project ID
        recaptcha_site_key: Site key obtained by registering a domain/app to use recaptcha services.
        token: The token obtained from the client on passing the recaptchaSiteKey.
            To get the token, integrate the recaptchaSiteKey with frontend. See,
            https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
        recaptcha_action: The action name corresponding to the token.
        hashed_account_id: Create hashedAccountId from user identifier.
            It's a one-way hash of the user identifier: HMAC SHA 256 + salt
        email: Email id of the user to trigger the MFA challenge.
        phone_number: Phone number of the user to trigger the MFA challenge.
```